### DISCUSSION TITLE
Complete Tutorial

### DISCUSSION THREAD

**Discussion by borislavmilijic** [advplyr/audiobookshelf#872] [Q&A] [answered]
Hi there! 
Could someone post a full detailed tutorial how to set up the Server part with Nginx Proxy Manager and connecting it with the app outside home network? Unfortunately the Readme doesn't provide enough details for me to able to set it up completely. 
I have the app running in docker, Nginx Proxy Manager with letsencrypt companion also running in docker and i have a domain for myself, but i am unable to connect the dots since not very Admin savvy. 
Thanks in advance!

Regards,
Borislav

**Comment by advplyr** [comment, marked as answer]
There are many knowledgeable people in the discord server who I'm sure will help you get set up
